plenum/server/database_manager.py

```diff
@@ -13,6 +13,7 @@ def __init__(self):
         self.stores = {}
         self.trackers = {}
         self._init_db_list()
+        self._state_version_controller = None
```

Comment:
Would it be better to call it in a more generic way: `txn_version_controller`? So that it can be potentially used not only for state versioning, but for other version-related things as well?